frontend/src/pages/DetailedReviewPage/components/DetailedReviewPageContents/styles.ts

```diff
@@ -1,12 +1,31 @@
 import styled from '@emotion/styled';
 
+import media from '@/utils/media';
+
 export const DetailedReviewPageContents = styled.div`
-  width: ${({ theme }) => theme.formWidth};
   border: 0.1rem solid ${({ theme }) => theme.colors.lightPurple};
   border-radius: ${({ theme }) => theme.borderRadius.basic};
+
+  margin-top: 2rem;
+
+  @media (max-width: 767px) {
+    width: 92%;
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    width: 80%;
+  }
+
+  @media (min-width: 1024px) {
```

Comment:
```suggestion
  export const DetailedReviewPageContents = styled.div`
  width:80% 
  @media (max-width: 767px) {
    width: 92%;
  }
```
이렇게 해도 되는데 @media (min-width: 1024px)로 width를 지정한 이유가 있을까요?

Reply:
[위] 크기를 덮어 쓰는 문제라 바다 말대로 반대로 하면 적용될 것 같네요!
미디어 쿼리를 사용할 때 항상 크기가 작은 것 -> 큰 것 순서대로 오름차순 나열을 해야 한다(Gxx 피셜)고 알고 있었어서 순서를 유지하되 더 촘촘히 범위를 설정해줘야 하는 줄 알았는데..알고보니 반대였군요 😂

지금 바다와 에프이가 개선된 media 유틸 이야기를 나눈 걸 다 보고 온 상태입니다. 거기서 나왔던 말대로, 미디어 쿼리에 and를 적용하면 특정 breakpoint 이하에 일괄적으로 적용할 때 중복 코드가 생기니까 내림차순 크기 정렬을 하는 것까지 이해했습니다! 이 부분은 새 유틸을 적용할 때 함께 고칠게요.

[아래]이대로라면 기본이 80%, 767px까지 92%이면 1024부터는 70%대신 다시 80%가 적용되는 게 아닌가요?
사실 제가 질문을 잘 이해하지 못했습니다 😅😅

Reply:
미디어 쿼리 지정할 때 큰 크기-> 작은 크기, 작은 크기-> 큰 크기로 두 가지로 방법으로 지정할 수 있어요.  (어떤 것을 선택할지에는 정답이 없어요. 단! 미디어 쿼리의 범위가 헷갈리지 않게 흐름을 유지하는게 좋아요. ) 
media가 max 기준으로 되어 있고, 데스크탑 기준으로 기존에 한 디자인 작업이 있기 때문에 큰 크기-> 작은 크기 흐름으로 뷰를 좁혀간다고 생각하면 쉬울 거에요

큰 크기에서 작은 크기 기준으로 좁혀다니까,
media.medium -> media.small 순으로 wdith를 지정한다면, min-width를 사용할 필요가 없어져요.  

media를 작은 크기에서 큰 크기로 갔기 때문에, 기존에 작업 했던 데스크탑 기준의 크기 설정을 유지하기 위해서 마지막에 min-width를 사용한 경우들이 보여서 이야기 나누고 싶었어요 😊

Reply:
오 당초 default 해상도가 데스크탑이었으니 큰 -> 작으로 간다는 흐름이 자연스럽네요!! 1024px도 그런 맥락에서 나온 이야기였군요 ㅎㅎㅎ
이번에 새 media 받고 수정할 때 내림차순으로 작업할게요~ 친절하게 알려주셔서 감사합니당ㅋㅋㅋㅋ